base/vulkanexamplebase.h

```diff
@@ -50,9 +50,59 @@
 #define GAMEPAD_BUTTON_R1 0x1005
 #define GAMEPAD_BUTTON_START 0x1006
 
+#if defined(__linux__) && !defined(__ANDROID__)
```

Comment:
The `#ifdef` checks follows the pattern that is already there in other files.

The `GAMEPAD_BUTTON_*` related code might very well be Android-specific, but I did not touch that code as part of this patch.